### PR TITLE
blackpaopao pledge

### PR DESCRIPTION
**Pull request by woshiwjw11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,30 @@
 # Agoric Validator Pledge
 
-_context: see Jan 28 [Validator Delegation Program](https://agoric.com/blog/announcements/agoric-validator-program/) announcement._
+I,jiawei wang hereby commit to the following work and support for the benefit of the Agoric Network and Community. If, for any reason, I do not meet these commitments then I understand that I may lose delegation staked to me.:
```
Comment (jeetraut) [inline README.md:3]:
```suggestion
[blackpaopao Explorer Link](https://main.explorer.agoric.net/validator/agoricvaloper1myk93jzj5n5adfp2d4pu28gjxcmn9v4ezlv795)
I,jiawei wang hereby commit to the following work and support for the benefit of the Agoric Network and Community. If, for any reason, I do not meet these commitments then I understand that I may lose delegation staked to me.:
```

**Review by jeetraut — CHANGES_REQUESTED** [review]
Add explorer link

**Comment by woshiwjw11** [comment]
thank you

**Comment by jeetraut** [comment]
Hi. Thanks for the contribution, but we need your README.md file to be in a subfolder. Take a look at https://github.com/Agoric/validator-profiles/pull/2 for an example where the README.md file is BlockShark/README.md.

We could try to resolve the conflicts in this branch / PR, but that would be kind of a hassle, so... can we ask you to just make another PR?

**Pull request closed without merging**